modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/entry.js

```diff
@@ -13,25 +13,25 @@
  */
 
 import {render, useThunk} from 'frontend-js-react-web';
+import PropTypes from 'prop-types';
 import React, {useReducer} from 'react';
 
 import {AppContext} from './AppContext';
 import DataSetDisplay from './DataSetDisplay';
 import ViewsContext, {viewsReducer} from './views/ViewsContext';
 
 const App = ({apiURL, appURL, portletId, ...props}) => {
-	const {
-		activeViewSettings: {name: activeViewName, visibleFieldNames = {}},
-		views,
-	} = props;
+	const activeViewName = props.activeViewSettings?.name;
```

Comment:
We can get `activeViewSettings` and `views` destructured from props instead use `props.view` etc